symtab_builder.cc

```diff
@@ -42,6 +44,13 @@ uintptr_t SymtabBuilder::AddSym(const Syminfo& sym) {
     return index;
 }
 
+// Returns true when the symbol specified with (name, soname, version) is
```

Comment:
How about

Returns true and fills `st_value` to `value_or_index`

?

Reply:
I see. It looks good.